cmd/controller/app/controllers.go

```diff
@@ -96,6 +96,15 @@ func addControllers(mgr manager.Manager, client k8s.Client, informerFactory info
 				ConfigOperator:  devopsClient,
 				ReloadCasCDelay: s.JenkinsOptions.ReloadCasCDelay,
 			}, s.JenkinsOptions)
+
+			// add PipelineRun controller
+			if err := (&pipelinerun.Reconciler{
+				Client: mgr.GetClient(),
+				Scheme: mgr.GetScheme(),
+				Log:    ctrl.Log.WithName("pipelinerun-controller"),
+			}).SetupWithManager(mgr); err != nil {
+				klog.Errorf("unable to create jenkins-pipeline-controller, err: %v", err)
```

Comment:
The caller will not be able to check if this function has errors. So, we should not `eat` the erorrs.

Reply:
> The caller will not be able to check if this function has errors. So, we should not `eat` the erorrs.

Thanks for reviewing. I will fix it.